simplified-books-controller/src/main/java/org/nypl/simplified/books/controller/BookBorrowTask.kt

```diff
@@ -35,13 +35,26 @@ import org.nypl.simplified.books.book_database.api.BookFormats
 import org.nypl.simplified.books.book_registry.BookRegistryType
 import org.nypl.simplified.books.book_registry.BookStatus
 import org.nypl.simplified.books.book_registry.BookStatusDownloadErrorDetails
-import org.nypl.simplified.books.book_registry.BookStatusDownloadErrorDetails.*
```

Comment:
Grr, these star imports need to die. Even with Android Studio configured never to add them, they still sometimes get added automatically.

Reply:
Yeah, these can be tricky. We'd had a flag set disable the "star-imports" check but I thought I'd take a shot at removing it.